Link to reprompt permissions

khorne3 opened the pull request:
https://trello.com/c/geq0QeKL/2319-handling-declined-authorization-permissions

@mpaktiti What are your thoughts on where I put the link to the re-prompt for permissions doc? Too hidden? Location too awkward?

Commit:
Link to reprompt permissions

mpaktiti commented:
@khorne3 What if you replaced the last item (`read more about client authentication`) in this list? https://auth0.com/docs/connections/social/37signals#next-steps The file is shared by all social connections from what I remember.

Commit:
Update links

mpaktiti committed:
merge items to shorten list

List was getting too big (we must use max up to 5 items as per guidelines). I removed the Lock/Auth0.js references and added a more generic one to a doc that lists all our libraries.

mpaktiti committed:
Merge branch 'master' into social-connections
